spec/factories/profiles.rb

```diff
@@ -37,11 +37,6 @@
       in_person_verification_pending_at { 15.days.ago }
     end
 
-    trait :fraud_pending_reason do
```

Comment:
this trait means simply TM has flagged a profile for review only, we haven't evaluated it yet